kernel/packages/shared/profiles/transformations/processServerProfile.ts

```diff
@@ -57,7 +57,7 @@ export function processServerProfile(userId: string, receivedProfile: any): Prof
     hasClaimedName:
       typeof receivedProfile.hasClaimedName === 'undefined' ? !!receivedProfile.name : receivedProfile.hasClaimedName,
     description: receivedProfile.description || '',
-    ethAddress: userId || 'noeth',
+    ethAddress: receivedProfile.ethAddress || 'noeth',
```

Comment:
Keep in mind that if there is a profile that is stored on the server, it will probably have the address as lower case. For instance: https://peer.decentraland.org/content/entities/profile?pointer=0xcd6989ccbe98eaa0fa209cbaf688279c107d1061

I'm not sure how to solve this case, other than ensure that the address is checksummed here using an external library (which more or less defeats the purpose of avoiding the calculation)

Reply:
I could apply the lowercase at that moment in time or just use userId in there

Reply:
Also, probably in the servers everything is mapped to `userId` right? since ethAddress can contain "noeth" and couldn't be used as a key